acme.sh

```diff
@@ -491,8 +491,14 @@ _post() {
     CURL="$CURL --dump-header $HTTP_HEADER "
     if [ "$needbase64" ] ; then
       response="$($CURL -A "User-Agent: $USER_AGENT" -X POST --data "$body" $url | _base64)"
+      if [ $? -gt 0 ] ;then
```

Comment:
so,  why not add a variable "ret" here,  and move this block to the end of this "_post()" function?